src/native/event_loop_group.c

```diff
@@ -45,8 +45,7 @@ jlong JNICALL
     if (result != AWS_OP_SUCCESS) {
         aws_event_loop_group_clean_up(elg);
         aws_mem_release(allocator, elg);
-        aws_jni_throw_runtime_exception(
-            env, "EventLoopGroup.event_loop_group_new: aws_event_loop_group_default_init failed");
+        aws_jni_throw_last_error(env, "EventLoopGroup.event_loop_group_new: aws_event_loop_group_default_init failed");
```

Comment:
Not necessarily required for this PR, but we should make the error messages match the current function names and be consistent. Some of them reference the old style, and some are just wrong.